components/board/commonsysfs/ioctl.go

```diff
@@ -0,0 +1,168 @@
+package commonsysfs
+
+// This file is heavily inspired by https://github.com/mkch/gpio
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"syscall"
+	"unsafe"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+
+	"go.viam.com/rdk/components/board"
+)
+
+const (
+	gpioGetLineHandleIoctl       = 0xc16cb403
```

Comment:
can you document where these come from?

Reply:
Now that we're using the other repo instead of reimplementing ourselves, they went away.